routes/extensionRequests.js

```diff
@@ -18,7 +19,7 @@ router.get("/:id", authenticate, authorizeRoles([SUPERUSER, APPOWNER]), extensio
 router.patch(
   "/:id",
   authenticate,
-  authorizeRoles([SUPERUSER, APPOWNER]),
+  skipAuthorizeRolesUnderFF(authorizeRoles([SUPERUSER, APPOWNER])),
```

Comment:
- why're we doing this?

Reply:
here the `skipAuthorizeRolesUnderFF` is invoked to skip the `authorizeRoles` middleware so that the end point is accessible to normal user for editing the extension request.

Reply:
- do we plan to remove the `skipAuthorizeRolesUnderFF` & `authorizeRoles` middleware when removing the feature flag?

Reply:
Yes that is the plan. I have created an issues and added in the PR description under additional notes.